app/docs/enterprise/0.31-x/oidc-google.md

```diff
@@ -52,7 +52,15 @@ $ curl -i -X PATCH http://kong:8001/apis/example/plugins/<OIDC plugin ID> \
   --data config.consumer_claim="email"
 ```
 
-Now, if a user logs into a Google account with the email `user@example.com`, Kong will apply configuration associated with the consumer `Yoda` to their requests. Note that while Google provides account emails, not all identity providers will. OpenID Connect does not have many required claims--the [only required user identity claim](http://openid.net/specs/openid-connect-core-1_0.html#IDToken) is `sub`, a unique subscriber ID. Many optional claims [are
-standardized](http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims), however--if a provider returns an `email` claim, the contents will always be an email address.
+Now, if a user logs into a Google account with the email `user@example.com`, Kong will apply configuration associated with the consumer `Yoda` to their requests. Note that while Google provides account emails, not all identity providers will. OpenID Connect does not have many required claims--the [only required user identity claim][oidc-id-token] is `sub`, a unique subscriber ID. Many optional claims [are
+standardized][oidc-standard-claims], however--if a provider returns an `email` claim, the contents will always be an email address.
 
 This also requires that clients login using an account mapped to some consumer, which may not be desirable (e.g. you apply OpenID Connect to a service, but only use plugins requiring a consumer on some routes). To deal with this, you can set the `anonymous` parameter in your OIDC plugin configuration to the ID of a generic consumer, which will then be used for all authenticated users that cannot be mapped to some other consumer.
+
+
+[add-certificate]: /docs/latest/admin-api/#add-certificate
+[google-oidc]: https://developers.google.com/identity/protocols/OpenIDConnect
+[google-create-credentials]: https://console.developers.google.com/apis/credentials
+[add-api]: /docs/enterprise/latest/getting-started/adding-your-api/
```

Comment:
Instead of referring to /latest/ here, it'd be more likely to be correct to refer to the matching version doc using the /{{page.kong_version}}/ technique - see an example of that at the bottom of https://raw.githubusercontent.com/Kong/getkong.org/master/app/docs/enterprise/0.31-x/getting-started/enabling-plugins.md